addons/br/nfse/extensions.go

```diff
@@ -0,0 +1,32 @@
+package nfse
+
+import (
+	"github.com/invopop/gobl/cbc"
+	"github.com/invopop/gobl/i18n"
+	"github.com/invopop/gobl/pkg/here"
+)
+
+// Brazilian extension keys required to issue NFS-e documents.
+const (
+	ExtKeyService = "br-nfse-service"
+)
+
+var extensions = []*cbc.KeyDefinition{
+	{
+		Key: ExtKeyService,
+		Name: i18n.String{
+			i18n.EN: "Service Code",
+			i18n.PT: "Código Item Lista Serviço",
+		},
+		Desc: i18n.String{
+			i18n.EN: here.Doc(`
+				The service code as defined by the municipality. Typically, one of the codes listed
+				in the Lei Complementar 116/2003, but municipalities can make their own changes.
+
+				For further details on the list of possible codes, see:
+
+				* https://www.planalto.gov.br/ccivil_03/leis/lcp/lcp116.htm
+			`),
+		},
```

Comment:
Is is possible to add a `Pattern` flag perhaps to validate the codes?

Reply:
I chose not to validate anything since every municipality can define their own list of codes without restrictions on the format. The list in LEI116 is commonly used, apparently, but there's no guarantee those will be the codes in the municipality.